packages/common/src/projects/_internal/ProjectUiSchemaEdit.ts

```diff
@@ -58,6 +58,22 @@ export const uiSchema: IUiSchema = {
             },
           },
         },
+        {
+          labelKey: "{{i18nScope}}.fields.combobox.label",
+          scope: "/properties/view/properties/combobox",
+          type: "Control",
+          options: {
+            control: "hub-field-input-combobox",
```

Comment:
I think you'll want to pass in `allowCustomValues` and `selectionMode` as options here as well:

```ts
{
  control: "hub-field-input-combobox",
  allowCustomValues: true,
  selectionMode: multiple
}
```

same for categories below